.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: build
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install packages
+      run: |
+        sudo apt-get -y install pandoc
+      if: matrix.python-version == 3.6
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
+        python -m pip install -e .[dev,gis]
+    - name: Lint
+      run: make lint
+      if: matrix.python-version == 3.6
+    - name: Test with pytest
+      run: |
+        pytest --cov xclim -m "not slow"
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Where is this token defined?

Reply:
It's generated on as-needed basis: https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token. Each job generates a token and once completed, that token is destroyed. Coveralls can therefore not accept more runs for this build once this build has run to completion.